src/AppWrapper.js

```diff
@@ -49,7 +49,7 @@ const AppWrapper = () => {
                         }
                         return (
                             <WindowDimensionsProvider>
-                                <App d2={d2} dataEngine={dataEngine} />
+                                <App />
```

Comment:
Should we pass empty `schemas` to the `D2Shim` above to avoid the schema request?

Reply:
As mentioned above, I don't think this is needed.

Reply:
But I don't think `d2` does anything by default with schemas.
Shouldn't we pass `[]` (or any schema we actually need) to avoid all the schemas from being fetched?

Reply:
The D2Shim receives the list of schemas here: https://github.com/dhis2/dashboard-app/pull/1441/files#diff-cc53afa34ba20baa7d10a5a4b9ef436dae14fc7d0609856e4b5aaaf1ac8429e9R44

Then, the d2 singleton (accessed via `useD2` or future `D2Shim`) has the schemas loaded in it.

Reply:
Right, sorry my bad, the relevant code was hidden in the diff here so I missed the schema part.